samples/bootloader/README.rst

```diff
@@ -256,18 +234,18 @@ Building and running
 
 .. include:: /includes/build_and_run.txt
 
-The bootloader sample should be included as a child image in a multi-image build, rather than being built stand-alone.
-While it is possible to build this sample by itself and merge it into other application images, this process is not officially supported by the |NCS|.
+NSIB should be included as a child image in a multi-image build, rather than being built stand-alone.
+While it is technically possible to build NSIB by itself and merge it into other application images, this process is not supported.
```

Comment:
```suggestion
The NSIB should be included as a child image in a multi-image build, rather than being built stand-alone.
While it is technically possible to build the NSIB by itself and merge it into other application images, this process is not supported.
```